### ISSUE TITLE
Nuget package, wishlist and additional feature request

### ISSUE THREAD

**Issue by avatart0ph** [PerplexInternetmarketing/Perplex-Umbraco-Forms#2]
Great work! I have looked at the wishlist in the documentation. 
I have created a [WCAG](https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines) datepicker for our client. I can send a similar version for this via pull request if I have time

Was wondering why this is not available in Nuget? Would be awesome to have it there so we can track updates.

Also, is it in your roadmap to creating a form picker data type we can configure it to select only certain forms via folder?

**Comment by PerplexDaniel** [comment]
Hi Denise,

We never got to actually publish this via NuGet and distribute it on Our Umbraco, but I agree it would have been better to publish this via NuGet as well. I am not sure if we get around to doing this too soon, but we'll look into it.

If you can get around to submitting your datepicker somehow we would love to have a look at it, it seems like a very useful addition. Thanks in advance!

With regards to a roadmap, this package should be integrated in Umbraco Forms in the near future, so we do not plan on adding a lot of new functionality ourselves anymore. However, when it's integrated in Umbraco Forms you can submit feature requests like yours directly to Umbraco HQ to pick up.

**Comment by p-m-j** [comment]
Hey, thanks for your work!

> this package should be integrated in Umbraco Forms in the near future

Any sort of timeline on this point?

**Comment by PerplexDaniel** [comment]
Hi Paul,

Thanks! 

> Any sort of timeline on this point?

There have not been many developments in this regard so it is unsure at this point whether it will happen at all. I cannot give you any timeline unfortunately. For now you'll just have to install the package.

**Comment by p-m-j** [comment]
any plans to add to nuget instead?

We have written off usage as it will be too much of a pain to manage across our various environments (without adding binaries to our git repos)

**Comment by PerplexDaniel** [comment]
Yes, that's definitely planned for "soon", i.e. max 2 months from now. We also prefer NuGet over committing binaries and other files into git so it's time this package also supports NuGet. Will definitely give it some priority soon.

**Comment by p-m-j** [comment]
Awesome, looking forward to it cheers

**Comment by PerplexDaniel** [comment]
@rustybox I recently looked into a NuGet package and a release candidate is now out to be tested:
https://www.nuget.org/packages/Perplex.Umbraco.Forms.On.Steroids/1.8.4-rc1

`Install-Package Perplex.Umbraco.Forms.On.Steroids -Version 1.8.4-rc1`

If you install from the UI, make sure to check the "Include prerelease" checkmark in Visual Studio for it to show up in search results. 

If you run into anything let me know.

**Comment by dnwhte** [comment]
@PerplexDaniel can you provide the source for the nuget package (1.8.4)?

**Comment by PerplexDaniel** [comment]
The NuGet package (1.8.4-rc.1) is basically built from the source in master except the version number is changed and we create the `PerplexInsertUmbracoForm` macro in code. The rest is the same. Never got around to actually releasing some final as this issue remained silent, but rc.1 should be usable.
